docs/run_examples.jl

```diff
@@ -0,0 +1,18 @@
+using Test
+
+@testset "Examples" verbose=true begin
+    examples_root = realpath(joinpath(@__DIR__, "../docs/src/examples"))
+    for name in readdir(examples_root)
+        dir = joinpath(examples_root, name)
+        script = joinpath(dir, "$name.jl")
+        ispath(script) || continue
```

Comment:
Well, maybe `ispath` is not so aptly named: it checks whether the given path corresponds to an existing file.
In this case, this will be true for all examples following the naming convention `src/examples/$NAME/$NAME.jl`. This allows avoiding half backed examples which are already committed to the repository, but aren't yet in a runnable state.

Reply:
Got it, thanks.